### DISCUSSION TITLE
Please remove IEnumerable requirement for Collection Initializers

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#5050] [General] [answered]
Hi C# Team,
Would you please remove the requirement that a class implement ```IEnumerable``` in order to utilize collection initializers?

It would be really nice to be able to do things like:
```
var H = new HashCode()
            {
                Start_At,
                End_At,
            };
```

**Comment by svick** [comment, marked as answer]
This is a duplicate of https://github.com/dotnet/csharplang/discussions/455.
